hnn_core/tests/test_network.py

```diff
@@ -62,23 +62,55 @@ def test_spikes():
     spiketimes = [[2.3456, 7.89], [4.2812, 93.2]]
     spikegids = [[1, 3], [5, 7]]
     spiketypes = [['L2_pyramidal', 'L2_basket'], ['L5_pyramidal', 'L5_basket']]
-    spikes = Spikes(times=spiketimes, gids=spikegids, types=spiketypes)
+    spikes = Spikes(times=spiketimes, gids=spikegids, types=spiketypes,
+                    tstart=0.1, tstop=98.4)
     spikes.plot_hist(show=False)
     spikes.write('/tmp/spk_%d.txt')
     assert spikes == read_spikes('/tmp/spk_*.txt')
     assert ("Spikes | 2 simulation trials" in repr(spikes))
+    assert spikes.mean_rates() == {
+        'L5_pyramidal': 5.08646998982706,
```

Comment:
do you mind explaining how you got this number given the numbers above?

Reply:
Perhaps a janky solution, it's manually computed with numpy. On the next commit I can just use `np.isclose()` if you agree.

Reply:
I've replaced this with an explicit calculation of the rate in `test_network.py` [here](https://github.com/jasmainak/hnn-core/blob/d69566307e86aaa886d80fb62b327e62f9d6f98d/hnn_core/tests/test_network.py#L143).

Do you think this is a good solution? One ugly alternative is looping through each dictionary and asserting `np.allclose()` on each pass.